js/src/wp-seo-admin-global.js

```diff
@@ -374,6 +449,8 @@
 				$link.parent().addClass( "active" );
 
 				openHelpCenterTab( $container, $( "#" + target ) );
+				// Trigger a custom event to check the scrollable tables size.
+				$( window ).trigger( "yoast-table-scrollable-check-size" );
```

Comment:
This is the part I'm not so happy about. It seems to me it doesn't gets triggered twice though. The thing is on the "Titles & Metas" page there are 12 tables. They're initially hidden so they have no computed size. Only when they become visible we can calculate their size.

How we can detect when a table becomes visible?

- it could be inside the first tab that gets visible when opening the Video Slideout, so the event gets triggered there a first time
- it could be inside the other tabs, that get visible when clicking on the left links `.wpseo-help-center-item > a` so the event gets triggered also on that click

Worth noting the JS that runs on the event doesn't do anything if there are no tables.

I definitely agree there should be a better way to do this. Maybe find a way to indicate which tabs contain a table? Any suggestions very welcome.